shared/common-adapters/index.native.js

```diff
@@ -24,6 +25,7 @@ import {
 export * from './index.js'
 
 export {
+  KeyboardAvoidingBox,
```

Comment:
I think you can do this a bit differently:

https://gist.github.com/MarcoPolo/fc5cd704c8766732e032c600e208e220

So you still get the typing but now you don't need to make `keyboard-avoiding-box{flow,native}`